tests/console/uninstall_pattern.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Execute SUT changes which should be permanent
+# Maintainer: Zaoliang Luo <zluo@suse.de>
+
+use base "consoletest";
+use testapi;
+use utils;
+use ipmi_backend_utils 'use_ssh_serial_console';
+use strict;
+
+sub run {
+    my $self = shift;
+    # let's see how it looks at the beginning
+    save_screenshot;
+    check_var("BACKEND", "ipmi") ? use_ssh_serial_console : select_console 'root-console';
+
+    # Installing a minimal system gives a pattern conflicting with anything not minimal
+    # Let's uninstall 'the pattern' (no packages affected) in order to be able to install stuff
+    script_run 'rpm -qi patterns-openSUSE-minimal_base-conflicts && zypper -n rm patterns-openSUSE-minimal_base-conflicts';
```

Comment:
Could you check if this is even necessary anymore on all currently supported systems? E.g. check for Leap 42.3, the oldest still supported openSUSE version

Reply:
checked this already. This is not necessary anymore on all currently supported systems: TW, Leap (inc. 42.3) and SLES 12 SP4. Will drop this part then.